python/repositories.bzl

```diff
@@ -30,9 +29,10 @@ py_repositories = _py_repositories
 python_register_multi_toolchains = _python_register_multi_toolchains
 python_register_toolchains = _python_register_toolchains
 
+# Useful for documentation.
```

Comment:
i.e. not intended to be public? Lets put a more explicit comment saying its only here for documentation, not public consumption, then.